src/Sylius/Bundle/CoreBundle/Resources/config/validation/ChannelPricing.xml

```diff
@@ -23,24 +23,39 @@
         </constraint>
         <property name="price">
             <constraint name="Range">
-                <option name="min">0.00</option>
+                <option name="min">0</option>
                 <option name="minMessage">sylius.channel_pricing.price.min</option>
                 <option name="groups">sylius</option>
             </constraint>
+            <constraint name="LessThan">
+                <option name="value">2147483647</option>
+                <option name="groups">sylius</option>
+                <option name="message">sylius.channel_pricing.price.max</option>
+            </constraint>
         </property>
         <property name="originalPrice">
             <constraint name="Range">
-                <option name="min">0.00</option>
+                <option name="min">0</option>
```

Comment:
Judging by the changed scenario, it seems we're missing tests for validating the min of original and minimum prices.